Algorithms/Sorters/External/Storages/IntInMemoryStorage.cs

```diff
@@ -0,0 +1,43 @@
+namespace Algorithms.Sorters.External.Storages
+{
+    public class IntInMemoryStorage : ISequentialStorage<int>
+    {
+        private readonly int[] storage;
+
+        public IntInMemoryStorage(int[] array) => storage = array;
+
+        public int Length => storage.Length;
+
+        public ISequentialStorageReader<int> GetReader() => new InMemoryReader(storage);
+
+        public ISequentialStorageWriter<int> GetWriter() => new InMemoryWriter(storage);
+
+        private class InMemoryReader : ISequentialStorageReader<int>
+        {
+            private readonly int[] storage;
+            private int offset;
+
+            public InMemoryReader(int[] storage) => this.storage = storage;
+
+            public void Dispose()
```

Comment:
Codacy found an issue: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6254880)